lib/vm/src/vmcontext.rs

```diff
@@ -336,6 +345,16 @@ unsafe impl Send for VMMemoryDefinition {}
 /// correctness in a multi-threaded context is concerned.
 unsafe impl Sync for VMMemoryDefinition {}
 
+impl MemoryUsage for VMMemoryDefinition {
+    fn size_of_val(&self, tracker: &mut dyn MemoryUsageTracker) -> usize {
+        if tracker.track(self.base as *const _ as *const ()) {
+            POINTER_BYTE_SIZE * (self.current_length as usize)
+        } else {
+            0
+        }
```

Comment:
Without context this reads as

"If we're not tracking anything at this address, then the size is 0, otherwise it's this size"

I realize it's the opposite, but it reads that way 😄

Reply:
I don't know if there's anything actionable to do here, just thought I'd share this feedback

Reply:
The name `track` doesn't express well what it does. Here is its documentation:

> When first called on a given address returns true, else returns false.

If you can come with a better name, I'll take it!